panoptes_aggregation/routes.py

```diff
@@ -69,6 +70,16 @@ def make_application():
                         static_url_path='',
                         static_folder='../docs/build/html')
     application.json_encoder = MyEncoder
+    CORS(
+        application,
+        origins=[
+            r'^https?:\/\/([a-z0-9-.]+zooniverse.org)',
+            r'^https?:\/\/(127\.0\.0\.1|localhost|[a-z0-9-]+\.local)(:\d+)?$'
+        ],
+        methods=[
+            'POST'
```

Comment:
Will you need OPTIONS here as well? It may not be but all preflight CORS requests are OPTIONS, I'm not sure

Reply:
I will add it in to be safe.